src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -307,19 +308,19 @@ public override void Write(byte[] buffer, int offset, int count)
                 Marshal.Copy(buffer, currentOffset, BufferIn, copyLen);
                 bytesRemain -= copyLen;
                 currentOffset += copyLen;
-                AvailIn = (IntPtr)copyLen;
+                AvailIn = (nuint)copyLen;
                 NextIn = BufferIn;
                 while ((int)AvailIn > 0)
                 {
                     if (!BrotliNative.BrotliEncoderCompressStream(_encoder.State, BrotliNative.BrotliEncoderOperation.Process, ref AvailIn, ref NextIn, ref AvailOut,
                         ref NextOut, out totalOut)) throw new System.IO.IOException("Unable compress stream"); 
```

Comment:
Regardless of having a custom exception, strings should be in a resource files. Take a look at how exceptions are thrown [here](https://github.com/dotnet/corefx/blob/master/src/System.IO.Compression/src/System/IO/Compression/GZipStream.cs)